media-gfx/openscad/openscad-2019.05-r1.ebuild

```diff
@@ -0,0 +1,82 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit elisp-common qmake-utils xdg
+
+SITEFILE="50${PN}-gentoo.el"
+
+DESCRIPTION="The Programmers Solid 3D CAD Modeller"
+HOMEPAGE="https://www.openscad.org/"
+SRC_URI="https://github.com/${PN}/${PN}/releases/download/${P}/${P}.src.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="emacs"
+
+# FIXME: add optional lib3mf
```

Comment:
what about lib3mf? can it be disabled at configure time so that it does not end up to be the same automagic dep [in case it gets introduced to the repository](https://github.com/gentoo/gentoo/pull/12097)?

Reply:
no it will need another revbump. I was not aware @salfter already prepared a PR.
So I would suggest to not merge this at this time. I will change the ebuild to also add lib3mf support and we wait for the lib3mf ebuild to be merged. IMO there's no need for another revbump in such a short time. What do you think?

Reply:
You're right with the `-r0` drop! I'll do that.

Reply:
The lib3mf is marked as REQUIRED in the CMakeLists.txt file, but it doesn't throw a error (I think due to QUIET passed to find_package), but only shows a message and even builds without it.